buildpack.go

```diff
@@ -29,6 +30,11 @@ func (bp Buildpack) noAPI() Buildpack {
 	return bp
 }
 
+func (bp Buildpack) withHomepage(bpInfo BuildpackInfo) Buildpack {
+	bp.Homepage = bpInfo.Homepage
```

Comment:
```suggestion
func (bp Buildpack) withHomepage(string url) Buildpack {
	bp.Homepage = bpInfo.Homepage
```

It seems unnecessary to pass a whole object in a method that only cares about a single value.

Reply:
Ah, good call!